src/pytest_mysql/executor.py

```diff
@@ -121,6 +121,7 @@ def shutdown(self):
             user=self.user
         )
         subprocess.check_output(shutdown_command, shell=True)
+        self._clear_process()
```

Comment:
Okay, calling clear_process here make the super's stop obsolete, so actually in this form, the shutdown should become stop itself.